subprojects/configuration-cache/build.gradle.kts

```diff
@@ -75,6 +61,7 @@ dependencies {
     implementation(project(":build-option"))
 
     implementation(libs.capsule)
+    implementation(libs.configurationCacheReport)
```

Comment:
`configuration-cache-report` is not an implementation dependency. The report template is extracted at build time and placed as a resource into `configuration-cache.jar`.

Reply:
Removed.